### PR TITLE
Add `@MavenTest` annotation to README example

### PR DESCRIPTION
**Pull request by Stephan202**
Looks like this annotation was accidentally dropped in eb8bb27ca8648b0254c2b4486cb153e8d23f86d2.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ Of course there is a lot of convenience integrated into it to make integration t
 The requirements to write integration tests with the integration testing framework are the folloing:
 
 * JDK11+
-* Apache Maven 3.1.0 or above. (I recommend to use Maven 3.8.1)
+* Apache Maven 3.1.0 or above. (I recommend to use Maven 3.8.2)
```
Comment (Stephan202) [inline README.md:66]: Unrelated change; let me know if you'd like to see this reverted. (Or perhaps rephrased to "I recommend using the latest release".)

**Comment by khmarbaise** [comment]
Yes that's unfortunately true. Thanks for finding that.

**Comment by khmarbaise** [comment]
Merged to master. Thanks for your contribution.
